atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -30,7 +32,60 @@
 @JsonTypeName(CassandraKeyValueServiceConfig.TYPE)
 @Value.Immutable
 public abstract class CassandraKeyValueServiceConfig implements KeyValueServiceConfig {
-    
+
+    @JsonIgnore
+    public static final CassandraKeyValueServiceConfig DEFAULT = new CassandraKeyValueServiceConfig() {
+        @Override
+        public Set<String> servers() {
+            return ImmutableSet.of("localhost");
+        }
+
+        @Override
+        public int port() {
+            return 9160;
+        }
+
+        @Override
+        public boolean ssl() {
+            return false;
+        }
+
+        @Override
+        public int replicationFactor() {
+            return 1;
+        }
+
+        @Override
+        public String keystore() {
+            return "./security/Server_Keystore";
+        }
+
+        @Override
+        public String keystorePassword() {
+            return "atlasserver";
+        }
+
+        @Override
+        public String truststore() {
+            return "./security/Client_Truststore";
```

Comment:
oh, nvm, I see what this is. I think it'd be better to use the builder to do the default thing:

    DEFAULT = ImmuatbleCassandraKeyValueServiceConfig.builder()
        .addServer("localhost")
        .port(9160)
        ...